bedevere/close_pr.py

```diff
@@ -0,0 +1,28 @@
+"""Automatically close PR that tries to merge maintenance branch into master"""
+import re
+
+import gidgethub.routing
+
+PYTHON_MAINT_BRANCH_RE = re.compile(r'^python:\d+.\d+$')
+
+router = gidgethub.routing.Router()
+
+@router.register("pull_request", action="opened")
+@router.register("pull_request", action="synchronize")
+async def close_invalid_pr(event, gh, *args, **kwargs):
+    """
+    Close the invalid PR
+    PR is considered invalid if:
+    base_label is 'python:master', and
+    head_label is 'python:<maint_branch>'
+    """
+    head_label = event["pull_request"]["head"]["label"]
+    base_label = event["pull_request"]["base"]["label"]
+
+    if PYTHON_MAINT_BRANCH_RE.match(head_label) and \
+        base_label == "python:master":
+        data = {'state': 'closed',
+                'maintainer_can_modify': True
```

Comment:
Either pull the closing brace up to this line, or move all dict lines on their own, have them all end with a comma, and have the closing brace be indented to the same level as `data`.